pkg/services/publicdashboards/publicdashboard.go

```diff
@@ -16,6 +16,7 @@ import (
 //go:generate mockery --name Service --structname FakePublicDashboardService --inpackage --filename public_dashboard_service_mock.go
 type Service interface {
 	FindPublicDashboardAndDashboardByAccessToken(ctx context.Context, accessToken string) (*PublicDashboard, *dashboards.Dashboard, error)
+	FindEnabledPublicDashboardAndDashboardByAccessToken(ctx context.Context, accessToken string) (*PublicDashboard, *dashboards.Dashboard, error)
```

Comment:
Instead of creating a new method, use the same method without checking enablement inside. The responsibility on take different actions for enabled/disabled pubdashes is on the caller.

Reply:
te caller is the api/controller itself for example. Do you think it's logic that should be in that layer? 
On the other hand, there might by many callers that will make the same comparison, so I thought it would be good to add this method

Reply:
I didn't see it in the controller, It should be fine then. I'm worried about having a lot of methods like this checking different attributes. At some point you can't reuse them.